doc/operator-scope.md

```diff
@@ -119,6 +119,8 @@ This scope is ideal for the cases where an instance(CR) of some Kind(CRD) will b
 
 ### Example for changing the CRD scope from namespace to cluster 
 
+The following example is for Go based-operators. Note that for Helm and Ansible based-operators the changes inn the CRD required to be done manually. 
+
```

Comment:
^
Add missing info in the operator scope doc to make clear that the steps to gen a cluster-scoped CR via the tool are valid just for Golang since the doc is the main menu